redis/tests/test_types.rs

```diff
@@ -93,6 +101,33 @@ fn test_u32() {
     }
 }
 
+#[test]
+fn test_parse_boxed() {
+    use redis::Value;
+
+    for parse_mode in [RedisParseMode::Owned, RedisParseMode::Ref] {
+        let simple_string_exp = "Simple string".to_string();
+        let v = parse_mode.parse_redis_value(Value::SimpleString(simple_string_exp.clone()));
+        assert_eq!(v, Ok(Box::new(simple_string_exp.clone())));
+    }
+}
+
+#[test]
+fn test_parse_arc() {
+    use redis::Value;
```

Comment:
I think we can move these using statements out of the tests, to the top of the file.